app/tests/e2e/test_verifier.py

```diff
@@ -182,6 +185,7 @@ async def test_reject_proof_request(
         bob_and_alice_connection["alice_connection_id"],
         protocol_version=ProofRequestProtocolVersion.v10.value,
     )
+    time.sleep(3)
```

Comment:
Ugh we reallyyyy need to get rid of those timeouts..... 

Webhooks FTW

Reply:
yeah will do that asap. Agreed. Slightly annoying.

Reply:
Just added this because without it worked about half of the time and errored out otherwise

Reply:
Yeah it's necessary at the moment 👍